File: deepseek/nivel avanzado/javascript/palindromo.js
function esPalindromo(texto) {
    const limpio = texto.toLowerCase().replace(/[^a-z0-9]/g, '');
    return limpio === limpio.split('').reverse().join('');
}

const cadena = prompt("Ingrese una cadena de texto:");
console.log("¿Es palíndromo?", esPalindromo(cadena));

File: deepseek/nivel avanzado/python/palindromo.py
import re

def es_palindromo(texto):
    texto = re.sub(r'[^a-zA-Z0-9]', '', texto).lower()
    return texto == texto[::-1]

cadena = input("Ingrese una cadena de texto: ")
print("¿Es palíndromo?", es_palindromo(cadena))